resources/quality/nwa3d_a5/nwa3d_a5_best.inst.cfg

```diff
@@ -12,118 +12,120 @@ global_quality = True
 
 [values]
 layer_height = 0.08
-initial_layer_height = 0.25
+layer_height_0 = 0.24
 line_width = 0.4
-initial_layer_line_width = 100
+wall_line_width_0 = 100
 wall_thickness = 0.8
-outer_wall_wipe_distance = 0.2
-top_surface_skin_layers = 1
+wall_0_wipe_dist = 0.2
+roofing_layer_count = 1
 top_bottom_thickness = 0.6
 top_thickness = 0.8
 top_layers = 5
 bottom_thickness = 0.6
 bottom_layers = 3
 top_bottom_pattern = lines
-bottom_pattern_initial_layer = lines
-outer_wall_inset = 0
+top_bottom_pattern_0 = lines
+wall_0_inset = 0
 optimize_wall_printing_order = false
-outer_before_inner_walls = false
-alternate_extra_wall = false
-compensate_wall_overlaps = true
-compensate_outer_wall_overlaps = true
-compensate_inner_wall_overlaps = true
-minimum_wall_flow = 0
-fill_gaps_between_walls = everywhere
+outer_inset_first = false
+alternate_extra_perimeter = false
+travel_compensate_overlapping_walls_enabled = true
```

Comment:
Please replace all the occurrences of `true` with `True`

Reply:
Just did, thanks!